docs/pages/blog/material-ui-v6-is-out.md

```diff
@@ -0,0 +1,228 @@
+  font: var(--mui-font-body1);
+}
+```
+
+## New method of applying styles
+
+The new API `theme.applyStyles` has been added for creating specific mode styles. It's designed to replace the `theme.palette.mode === 'dark'` condition and to work with Pigment CSS.
+
+The snippet below will work for both Emotion and Pigment CSS.
+
+```jsx
+const StyledInput = styled(InputBase)(({ theme }) => ({
+  padding: 10,
+  width: '100%',
+  borderBottom: `1px solid #eaecef`,
+  ...theme.applyStyles('dark', {
+    borderBottom: '1px solid #30363d',
+  })
+  '& input': {
+    borderRadius: 4,
+    backgroundColor: '#fff',
+    ...theme.applyStyles('dark', {
+      backgroundColor: '#0d1117',
+    })
+  },
+}));
+```
+
+## Container Queries
+
+We added [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries) utility based on the existing `theme.breakpoints` API.
+
+This feature lets you define styles based on the parent container's width instead of the viewport.
+
+```jsx
+const Component = styled('div')(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(2),
+  [theme.containerQueries.up('sm')]: {
+    flexDirection: 'row',
+  },
+  [theme.containerQueries('sidebar').up('400px')]: {
+    flexDirection: 'row',
+  },
+}));
+```
+
+It also works with the `sx` prop:
+
+```jsx
+<Card
+  sx={{
+    '@sm': {
+      flexDirection: 'row',
+    },
+    '@400/sidebar': {
+      flexDirection: 'row',
+    },
+  }}
+/>
+```
+
+## Stabilized Grid v2
+
+Grid v2 has been stabilized and is now using CSS gap for creating space between grid items.
+The responsive configuration has been consolidating into `size` and `offset` props for consistency.
+
+```jsx
+import Grid from '@mui/material/Grid2';
+
+<Grid container>
+  <Grid size={{ xs: 6, sm: 4, lg: 3 }} />
+  <Grid size={{ xs: 6, sm: 4, lg: 3 }} />
+  <Grid size={{ xs: 6, sm: 4, lg: 3 }} />
+</Grid>;
+```
+
+## Revamping the free templates
+
+Explore the new and enhanced [Material UI free templates](https://mui.com/material-ui/getting-started/templates/) to see these amazing features in action. We've fully revamped the templates to provide the perfect starting point for your project, whether you're adding sleek styles or using the template's sections.
+
+{{"component": "components/blog/material-ui-v6-is-out/FreeTemplatesBento.js"}}
+
+### Custom syles
+
+The new custom styles we've added allow you to dive into the best approaches for customization or simply copy and paste some stylish elements.
+
+{{"component": "components/blog/material-ui-v6-is-out/CustomThemeComparison.js"}}
+
+Stay tuned, as many more updates are coming in the upcoming months! We'd love to hear your feedback on which templates or sections you'd like to see next.
+
+For your full-stack needs you can also explore [Toolpad Core](https://mui.com/toolpad/)—a Next.js starter for building dashboards. It offers components like dashboard layout, next-auth powered sign-in page and more.
```

Comment:
@prakhargupta1 I would imagine us adding a small subsection within the "Revamping the free templates" section, perhaps titled "Full stack apps", with an image similar to the one used in the "Custom styles" subsection - a composite of all the components we have available


```suggestion
### Full Stack Apps
To get started building full-stack apps fast, we're building [Toolpad Core](https://mui.com/toolpad) - get  Material UI, Next.js, routing, layout components and authentication out of the box while you build dashboards or internal tools.
```

I would place this subsection before the "Stay tuned, as many more..." line

Reply:
I think a small image should be fine, given we're at the end of the page. People might skim the text entirely if there's no visual attention catcher.